fuzzers/060-bram-cascades/generate.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Can this use the general utils generate.sh? I'm not sure if I see anything special here other than the grep? And that looks like just for logging

Reply:
ah I see theres actually a different print scheme, nvm